features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Environments/Environments.jsx

```diff
@@ -292,7 +292,7 @@ const useStyles = makeStyles((theme) => ({
 export default function Environments() {
     const classes = useStyles();
     const theme = useTheme();
-    const { maxCommentLength } = theme.custom;
+    const { maxCommentLength } = "255";
```

Comment:
This is Eslint issue, it should be a single quote. also please check line 294 as well, without theme.custom we could remove that as well.